ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -333,7 +333,18 @@ default_interfaces:
 < node_type_key >:
 
   defaults:
-    # node BGP AS | Required.
+    # Node BGP AS | Required.
+    # For eBGP topologies, bgp_as can be defined as a range of ASNs under defaults.  This range will be expanded and ASNs allocated
```

Comment:
It would accept the range anywhere in the hierarchy.
```suggestion
    # For eBGP topologies, bgp_as can be defined as a range of ASNs.  This range will be expanded and ASNs allocated
```

Reply:
Done, thanks.